### PR TITLE
Minor re-org and spelling fixes 

### PR DESCRIPTION
**Pull request by cansavvy**
We had some discussion about what should be covered in this website. That led to this first PR: #6 but then #8 left some gnarly merge conflicts. So this is the newer version of #6

I've tried to keep what I felt were the important points from the Slack and BaseCamp discussions and rectify that with what is here. But honestly we just need a thing merged.  

### Changes/adds in this PR
The categories of "support", "training", and "community" weren't distinct to me. It wasn't clear to me what was on each of those pages until I went there. So I tried to make the names slightly more clear to someone about what was where. But I know Amy linked things to the links she made so I tried not to change them more than needed. 

So now, support is still support but "community" -> "Hutch Data Groups" and "training" is "training resources". To me, training could mean events, groups, links, or something else. And community to me would be where I'd go to connect with the community but in this case it looks like that is what "support" is supposed to be. Furthermore, I put ScWiki on the pages with the resources. 

### The new organization proposed by this PR: 

- `Support` = where people go to connect to support. It includes the Slack, The Calendar, as well as where we will post Office Hours once we've established them. 
- `Training Resources` = Where people can find links to training things that the fhdasl supports. Things like AnVIL, ITN, JH data courses, DataTrail, and SciWiki 
- `Hutch Data Groups`  = Where you can find the groups, tools and platforms that do data work at the Hutch.

### CODE REVIEW

**Comment by cansavvy** [comment]
This is not 100% of the way there, but it gets us closer.

File: _pages/training.md [hunk 1]
```diff
@@ -1,31 +1,21 @@
...
-
-
-
 ## DaSL Training Resources
-FH DaSL staff have developed many training resources as part of ongoing and prior collaborations and efforts.  Various resources spanning a wide range of data science and tool-specific topics taht we have previously developed are available from the sources listed below.  
+FH DaSL staff have developed many training resources as part of collaborations and efforts.  Various resources spanning a wide range of data science and tool-specific topics that we have previously developed are available from the sources listed below. For a longer list of recommended resources (not just ones supported by us, [go here](https://hutchdatascience.org/code_review/more_resources.html)).
 
 (in alphabetical order)
 
 ### AnViL
```
Comment (avahoffman) [inline _pages/training.md:11]:
```suggestion
### AnVIL
```

File: _pages/training.md [hunk 2]
```diff
@@ -1,31 +1,21 @@
...
 [AnVIL](https://anvilproject.org/learn) is a computing platform that enables researchers to analyze controlled access data sets in a cloud computing environment. It has loads of training materials to support those using it!
 
 
+### Code Review Guidance for Research Labs
+Leading a lab with novice or experienced code writers and users?  Either way, see our [Code Review](/code_review/) materials that include helpful suggestions for various types of lab members, expertise and group dynamics.
```
Comment (avahoffman) [inline _pages/training.md:16]: This feels like a more specific resource than the others, which are platforms/organizations. We should probably think about where our specific instructionals (like this, or intro to R) live. Maybe we could have like a "quick start training" tab or section? That way folks know they're jumping right into a tutorial

**Comment by avahoffman** [comment]
Minor comments but seems good to me! We can always update again later and do smaller bite sized PRs to avoid those merge headaches :)

**Review by avahoffman — APPROVED** [review]

File: _pages/training.md (hunk 2 above)
Comment (cansavvy) [reply _pages/training.md:16]: Yeah. I don't expect this to be its permanent home but until we have other things like it, it's hard to know where it fits without us promising we are for sure going to make more things like it. Does that make sense?

**Pull request merged**